modules/runner/src/testFixtures/java/org/apache/ignite/internal/Cluster.java

```diff
@@ -231,31 +235,30 @@ private void startAndInit(
 
         TestIgnitionManager.init(metaStorageAndCmgNodes.get(0), builder.build());
 
-        for (IgniteServer node : nodes) {
-            assertThat(node.waitForInitAsync(), willCompleteSuccessfully());
+        for (ServerRegistration registration : nodeRegistrations) {
+            assertThat(registration.registrationFuture(), willCompleteSuccessfully());
         }
 
         started = true;
     }
 
     /**
-     * Starts a cluster node with the default bootstrap config template and returns its startup future.
+     * Starts a cluster node with the default bootstrap config template and returns it and its registation future.
```

Comment:
```suggestion
     * Starts a cluster node with the default bootstrap config template and returns it and its registration future.
```


And "return" part of javadoc is not changed